Backend/src/config/passport-setup.ts

```diff
@@ -0,0 +1,61 @@
+import passport from 'passport';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+import { type users } from '@prisma/client'; // Make sure to import the User type from Prisma

```

Comment:
why `type users` ?

Reply:
Because after completing the process we want to send users to the cookie, so I made an import for users
Is there a better way to import users via typescript?